docs/pages/workflow/customizing.mdx

```diff
@@ -60,3 +60,10 @@ When you're ready to ship your app, you can [build it with EAS Build](/build/int
   cmd={['# Install the CLI', '$ npm i -g eas-cli', '', '# Build your app!', '$ eas build -p all']}
   cmdCopy="npm i -g eas-cli && eas build -p all"
 />
+
+
+## Creating native modules
+
+The [Expo Module API](/modules/overview) enables developers to build modules for Expo and React Native projects using Swift, Kotlin, and TypeScript. We use it for most modules in the [Expo SDK](/versions/latest). [Learn more about the Expo Module API design considerations](/modules/overview).
+
+Another option is to use [React Native's Core Native Modules API](https://reactnative.dev/docs/native-modules-intro), which may require some C++ knowledge in addition to Objective C and Java. Most React Native modules in the ecosystem are built using this API because it is and always has been part of React Native, while the Expo Module API is new and intended to solve for many of the pain points of using the core API.
```

Comment:
```suggestion
Another option is to use [React Native's Core Native Modules API](https://reactnative.dev/docs/native-modules-intro), which may require some C++ knowledge in addition to Objective-C and Java. Most React Native modules in the ecosystem are built using this API because it is and always has been part of React Native, while the Expo Module API is new and intended to solve for many of the pain points of using the core API.
```

Reply:
Maybe we can add this to the dict?